defusedxml/ElementTree.py

```diff
@@ -29,7 +29,7 @@
 __origin__ = "xml.etree.ElementTree"
 
 def _get_py3_cls():
-    """Python 3.3 hides the pure Python code but defusedxml requires it.
+    """Python 3.3+ hide the pure Python code but defusedxml requires it.
```

Comment:
The findings you detail in the commit message actually say the opposite of this comment change.